app/assets/stylesheets/profile-preview-card.scss

```diff
@@ -8,9 +8,33 @@
 
   &__content.crayons-dropdown:hover {
     display: block;
+    animation: hoverAppear 500ms;
+
+    &.showing {
+      animation: none;
+    }
   }
 
   &__trigger:hover + .profile-preview-card__content.crayons-dropdown {
     display: block;
+    animation: hoverAppear 500ms;
+
+    &.showing {
+      animation: none;
+    }
```

Comment:
This is to make sure that if a preview card is fully opened (by clicking on it), we don't play this animation if a user then hovers on the author name (which would result in the card disappearing and reappearing again)